engine/src/main/java/org/hibernate/validator/internal/metadata/core/ConstraintHelper.java

```diff
@@ -201,7 +200,7 @@ public ConstraintHelper() {
 
 		putConstraints( tmpConstraints, Digits.class, DigitsValidatorForCharSequence.class, DigitsValidatorForNumber.class );
 
-		List<Class<? extends ConstraintValidator<Future, ?>>> futureValidators = newArrayList( 11 );
+		List<Class<? extends ConstraintValidator<Future, ?>>> futureValidators = new ArrayList<>( 18 );
```

Comment:
That's an array list, not a set with a load factor (assuming that's what you have in mind).

I'd prefer to phase out `newArrayList() ` as it provides no advantage any longer as we can rely on diamond. It's different for `newHashSet() `.